save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestExecutionController.kt

```diff
@@ -22,8 +24,8 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
     fun saveTestResult(@RequestBody testExecutions: List<TestExecution>): ResponseEntity<String> {
         try {
             testExecutionService.saveTestResult(testExecutions)
-        } catch (exception: IllegalArgumentException) {
-            return ResponseEntity.ok().body("Error to save")
+        } catch (exception: DataAccessException) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Error to save")
```

Comment:
I'd rather say it is internal server error. Do we have situation when it can be bad request? When test execution id is not present in DB? Then this situation should be separated from the situation when there are other problems with DB. I believe spring data throws different subtypes of `DataAccessException` in these cases